analyzer/src/main/kotlin/managers/PIP.kt

```diff
@@ -74,6 +74,19 @@ object VirtualEnv : CommandLineTool {
     override fun getVersionRequirement(): Requirement = Requirement.buildIvy("15.1.+")
 }
 
+object ProjectVersion : CommandLineTool {
+    override fun command(workingDir: File?) = if (OS.isWindows) "python" else "python3"
+
+    fun isPython3(workingDir: File): Boolean{
+        val scriptFile = File.createTempFile("python_compatibility",".py" )
+
+        scriptFile.writeBytes(javaClass.classLoader.getResource("python_compatibility.py").readBytes())
+        val scriptCmd = run(scriptFile.absolutePath, "-d", workingDir.absolutePath)
```

Comment:
Please wrap this in `try` / `finally` and delete the temporary file in the the `finally` block. For an example see https://github.com/heremaps/oss-review-toolkit/blob/master/analyzer/src/main/kotlin/managers/Bundler.kt#L185-L192.

Reply:
I wrapped also second script (python_path.py). 

- [x] Delete file after processed.